plaso/parsers/utmp.py

```diff
@@ -162,13 +162,22 @@ def ParseFileObject(self, parser_mediator, file_object):
           'Unable to parse first utmp entry with error: {0!s}'.format(
               exception))
 
+    if not timestamp:
+      raise errors.UnableToParseFile(
+          'Unable to parse first utmp entry with error: missing timestamp')
+
+    if not event_data.hostname:
+      raise errors.UnableToParseFile(
+          'Unable to parse first utmp entry with error: missing hostname')
+
+    if not event_data.terminal:
+      raise errors.UnableToParseFile(
+          'Unable to parse first utmp entry with error: missing terminal')
+
     if not event_data.username:
```

Comment:
Please add check to see if `type of login != DEAD_PROCESS`

Reply:
Done.